final-project/scripts/main.js

```diff
@@ -1,10 +1,5 @@
 import "../styles/main.scss";
+import { BlogCarousel } from "./blog-carousel";
 
-// testing that `const` and arrow functions transpile correctly
-const before = "Hello -world!";
-const after = before
-  .split("")
-  .filter(char => char !== "-")
-  .join("");
-console.log({ before });
-console.log({ after });
+const blogCarousel = new BlogCarousel();
+blogCarousel.init(document.querySelector(".resources__carousel"));
```

Comment:
I know that this class wasn't used for styling purposes, but just as a recommendation it is better to use different naming for things manipulated by javascript: ie. `.js-resources__carousel`